demos/python_demos/human_pose_estimation_3d_demo/modules/parse_poses.py

```diff
@@ -14,7 +14,12 @@
 import numpy as np
 
 from modules.pose import Pose, propagate_ids
-from pose_extractor import extract_poses
+try:
+    from pose_extractor import extract_poses
+except ImportError as err:
+    raise ImportError("{}.\nModule 'pose_extactor' not found. "
+                      "Please build module according to documentation before usage.".format(err.msg))
```

Comment:
```suggestion
    raise ImportError("Module 'pose_extractor' not found. "
                      "Please build module according to documentation before usage.") from err
```

This should create a good error output without the need to manually compose the messages.